apps/web/src/pages/integrations/UpdateProviderPage.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import styled from '@emotion/styled';
+import { useMantineColorScheme } from '@mantine/core';
+import { colors } from '../../design-system';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useEffectOnce } from '../../hooks';
+import { useProviders } from './useProviders';
+import { IIntegratedProvider } from './IntegrationsStoreModal';
+
+export function UpdateProviderPage() {
+  const [selectedProvider, setSelectedProvider] = useState<IIntegratedProvider | null>(null);
+  const { providers } = useProviders();
+  const { colorScheme } = useMantineColorScheme();
+  const isDark = colorScheme === 'dark';
+  const { integrationId } = useParams();
+  const navigate = useNavigate();
+
+  useEffectOnce(() => {
+    const foundProvider = providers.find((provider) => provider.integrationId === integrationId);
+
+    if (foundProvider) {
+      setSelectedProvider(foundProvider);
+    }
+  }, integrationId !== undefined && providers.length > 0);
+
+  if (selectedProvider === null) {
+    return null;
+  }
+
+  return <SideBarWrapper dark={isDark}></SideBarWrapper>;
+}
+
+const SideBarWrapper = styled.div<{ dark: boolean }>`
```

Comment:
can we use theme here instead of passing 'dark'?

Reply:
It is in a different pr for the Update sidebar